### PR TITLE
Prepare 2021Q4 release

### PR DESCRIPTION
**Pull request by fpetrogalli**
* [ ] If an issue reporting the bug exists, I have mentioned it in the
      PR (do not bother creating the issue if all you want to do is
      fixing the bug yourself).
* [X] I have added/updated the `SPDX-FileCopyrightText` lines on top
      of any file I have edited. Format is `SPDX-FileCopyrightText:
      Copyright {year} {entity or name} <{contact informations}>`
      (Please update existing copyright lines if applicable. You can
      specify year ranges with hyphen , as in `2017-2019`, and use
      commas to separate gaps, as in `2018-2020, 2022`).
* [X] I have updated the `Copyright` section of the sources of the
      specification I have edited (this will show up in the text
      rendered in the PDF and other output format supported). The
      format is the same described in the previous item.
* [X] I have run the CI scripts (if applicable, as they might be
      tricky to set up on non-*nix machines). The sequence can be
      found in the [contribution
      guidelines](../CONTRIBUTING.md#continuous-integration). Don't
      worry if you cannot run these scripts on your machine, your
      patch will be automatically checked in the Actions of the pull
      request.
* [X] The pull request is done against the branch `next-release`.

### CODE REVIEW

File: main/acle.md [hunk 1]
```diff
@@ -1104,9 +1108,10 @@ This macro is defined for AArch32 only.
 ### 32-bit SIMD instructions
 
 `__ARM_FEATURE_SIMD32` is defined to 1 if the 32-bit SIMD instructions
-are supported and the intrinsics defined in [ssec-32SIMD](#simd-intrinsics-for-32-bit-operations) are
-available. This also implies support for the GE global flags which
-indicate byte-by-byte comparison results.
+are supported and the intrinsics defined in
+[ssec-32SIMD](#32-bit-simd-operations) are available. This also
```
Comment (ValeriaDaneva) [inline main/acle.md:1112]: Wouldn't this section link not work when  the document is converted to pdf?

File: main/acle.md [hunk 2]
```diff
@@ -2700,7 +2705,7 @@ operations [[G.191]](#G.191):
 This example assumes the implementation preserves the Q flag on return
 from an inline function.
 
-## SIMD intrinsics for 32-bit Operations
+## <span id="32-bit-simd-operations">32-bit SIMD Operations</span>
```
Comment (ValeriaDaneva) [inline main/acle.md:2708]: Does using `<span>` work after pdf conversion?

File: morello/morello.md [hunk 3]
```diff
@@ -159,6 +159,10 @@ All content in this document is at the **Alpha** quality level.
 
 * Update the copyright statement in section [Copyright](#copyright).
 * Add a reference to the [CHERI-HYBRID](https://github.com/CTSRD-CHERI/cheri-hybrid-c-guide) *CHERI Hybrid C/C++ Programming Guide* in section [Scope](#scope).
+* Converted document sources from reStructuredText (`.rst`) to
+  Markdown (`.md`). The [`pandoc`](https://pandoc.org/) is now used to
```
Comment (ValeriaDaneva) [inline morello/morello.md:163]:
```suggestion
  Markdown (`.md`). [`Pandoc`](https://pandoc.org/) is now used to
```
This suggestion applies for every instance this change is described. Alternatively, "The pandoc command" should also work.

File: main/acle.md (hunk 1 above)
Comment (fpetrogalli) [reply main/acle.md:1112]: Ah - good catch - the hyperlink was taking to the entry of the section in the TOC, not in the body of the specs. I have moved the `span` right before the sectioning, now it works. You can check in the artifacts of the latest build.

File: main/acle.md (hunk 2 above)
Comment (fpetrogalli) [reply main/acle.md]: Nope, see my previous comment in https://github.com/ARM-software/acle/pull/153#discussion_r778448755

File: morello/morello.md (hunk 3 above)
Comment (fpetrogalli) [reply morello/morello.md]: I opted for `The tool pandoc ...`

**Review by ValeriaDaneva — APPROVED** [review]

**Pull request merged**